docs/source/fundamentalConcepts.rst

```diff
@@ -112,11 +112,19 @@ units has `not ended well in the past <https://en.wikipedia.org/wiki/Mars_Climat
 Finally, having convinced you to think of images as objects occupying a physical region
 in space, we need to answer two questions:
 
-1. How do you access the pixel values in an image:
+1. How do you access the pixel values in an image?
 
-   .. code-block:: python
+   a. In dynamically typed typed languages such as Python and R use a type agnostic function:
+
+      .. code-block:: python
+
+        image.GetPixel((0,0))
+
+   b. In statically typed languages such as C# and C++ use type specific functions:
+
+      .. code-block:: C++
 
-     image.GetPixel((0,0))
+        image.GetPixelAsUInt8( std::vector<uint32_t>( 0, 0 ) )
```

Comment:
The new C++11 way is `image.GetPixelAsUInt8( {0, 0} )`.

Reply:
Well, new and shiny is always better. Will modify.

Reply:
Well, new and shiny is always better. Will modify.